service/src/test/java/greencity/mapping/service/GetServiceDtoMapperTest.java

```diff
@@ -0,0 +1,30 @@
+package greencity.mapping.service;
+
+import greencity.ModelUtils;
+import greencity.dto.service.GetServiceDto;
+import greencity.entity.order.Service;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class GetServiceDtoMapperTest {
+    @InjectMocks
+    private GetServiceDtoMapper mapper;
+
+    @Test
+    void convert() {
+        Service service = ModelUtils.getService();
+        GetServiceDto dto = ModelUtils.getServiceDto();
+        GetServiceDto convertDto = mapper.convert(service);
+
+        Assertions.assertEquals(dto.getId(), convertDto.getId());
```

Comment:
you convert `Service` to `GetServiceDto` (`service` -> `convertDto`)
Please check that `convertDto` contains the same values as `service`

Reply:
fixed